rpc_jobs/rpc_aio.yml

```diff
@@ -184,17 +184,6 @@
               maas = load 'pipeline_steps/maas.groovy'
               kibana = load 'pipeline_steps/kibana.groovy'
             }}
-            // We need to checkout the rpc-openstack repo on the CIT Slave
-            // so that we can cater for builds on the more restricted images
-            // used for artifact-based builds.
-            if (env.STAGES.contains("Upgrade")) {{
-              common.prepareRpcGit(env.UPGRADE_FROM_REF, env.WORKSPACE)
-            }} else {{
-              common.prepareRpcGit("auto", env.WORKSPACE)
-            }} // if
-            if(common.is_doc_update_pr("${{env.WORKSPACE}}/rpc-openstack")){{
```

Comment:
Ah, I see that now. I was effectively trying to revert https://github.com/rcbops/rpc-gating/commit/253cff09c8ac2601c5203b8a8a246cbf75442987 but I see now that the use of the CIT clone has evolved for this job.